qcengine/procedures/berny.py

```diff
@@ -0,0 +1,72 @@
+import logging
+import traceback
+from io import StringIO
+from typing import Any, Dict, Union
+
+import numpy as np
+import pkg_resources
+from qcelemental.models import OptimizationInput, OptimizationResult
+from qcelemental.util import which_import
+
+import qcengine
+
+from ..config import TaskConfig
+from .model import ProcedureHarness
+
+
+class BernyProcedure(ProcedureHarness):
+    _defaults = {"name": "Berny", "procedure": "optimization"}
+
+    def found(self, raise_error: bool = False) -> bool:
+        return which_import(
+            "berny", return_bool=True, raise_error=raise_error, raise_msg="Please install via `pip install pyberny`.",
+        )
+
+    def build_input_model(self, data: Union[Dict[str, Any], "OptimizationInput"]) -> "OptimizationInput":
+        return self._build_model(data, OptimizationInput)
+
+    def compute(self, input_data: "OptimizationInput", config: "TaskConfig") -> "OptimizationResult":
+        try:
+            import berny
+        except ModuleNotFoundError:
+            raise ModuleNotFoundError("Could not find Berny in the Python path.")
+        log_stream = StringIO()
+        log = logging.getLogger(f"{__name__}.{id(self)}")  # per-instance logger
+        log.addHandler(logging.StreamHandler(log_stream))
+        log.setLevel("INFO")
+        input_data = input_data.dict()
```

Comment:
Can you give some whitespace lines here and a few comments similar to other modules?

Reply:
Done.